constructor-manager-cli/setup.cfg

```diff
@@ -0,0 +1,60 @@
+[metadata]
+name = constructor-manager-cli
+version = 0.0.1
+description = Constructor environment and updates manager command line interface
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/napari/packaging/constructor-manager-cli
+author = napari team
+author_email = napari-steering-council@googlegroups.com
+license = MIT
+license_files = LICENSE
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Framework :: napari
```

Comment:
We can have a separate discussion on which packages are intended to be a napari plugin and should keep this classifier

Reply:
Typo, removing from files